supabase/seed/seed.mts

```diff
@@ -72,13 +72,23 @@ async function generateSeed(): Promise<void> {
                 generateFamily(
                     { min: 1, max: 8 },
                     {
-                        age: (ctx) => copycat.int(ctx.seed, { min: 0, max: 100 }),
+                        birthYear: (ctx) => copycat.int(ctx.seed, { min: 1901, max: 2024 }),
```

Comment:
Should we use current year for max and current year - 120 (?)?

Reply:
I've tried to use getCurrentYear() in seed.mts but it has an import error when I try to run generate seed. Can't seem to find a way to do this - any suggestions?

Reply:
Guess you can't import that function but what about just using what's inside of it i.e. just `new Date().getFullYear()`?

Reply:
Done :)